crates/vm2/src/single_instruction_test/state_to_zk_evm.rs

```diff
@@ -86,7 +86,7 @@ fn vm2_frame_to_zk_evm_frames<T, W>(
         heap_bound: frame.heap_size,
         aux_heap_bound: frame.aux_heap_size,
         total_pubdata_spent: PubdataCost(0),
-        stipend: frame.stipend,
```

Comment:
They forgot to remove the field? Or does the zk_evm dependency need updating?

Reply:
I've postponed updating the `zk_evm` dependency until it's released, yes.